lib/ItemRow/ItemRowAside.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { node } from 'prop-types';
+
+const ItemRowAside = ({ children, ...rest }) => {
+  return (
+    <div className="item-row__aside" {...rest}>
```

Comment:
Could this be an `aside` element?

Reply:
I did originally but the aside isn't in anything substantial to warrant it. For example `article` or whatever.